java/client/src/main/java/glide/api/models/commands/SortBaseOptions.java

```diff
@@ -0,0 +1,107 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.GenericClusterCommands;
+import glide.api.commands.GenericCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Optional arguments to {@link GenericClusterCommands#sort(String, SortClusterOptions)}, {@link
+ * GenericClusterCommands#sortReadOnly(String, SortClusterOptions)}, {@link
+ * GenericClusterCommands#sortStore(String, String, SortClusterOptions)}, {@link
+ * GenericCommands#sort(String, SortOptions)}, {@link GenericCommands#sortReadOnly(String,
+ * SortOptions)}, and {@link GenericCommands#sortStore(String, String, SortOptions)}
```

Comment:
nit: this is why I don't like linking to commands.  IntelliJ has a handy mechanic to "Find Usages" while listing everything here requires that we maintain this.  

You may remove these links if you'd like and just put "sort, sortReadOnly, and sortStore commands" - without the `{@link `